src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,35 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> wordsArray = new ArrayList<>();
+        StringBuilder sb = new StringBuilder(" ");
+        File file = new File(fileName);
+        if (file.length() == 0) {
+            return new String[0];
+        }
+        String line = "";
+        try {
+            FileReader fileReader = new FileReader(file);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            while ((line = bufferedReader.readLine()) != null) {
+                sb.append(line);
+            }
+            bufferedReader.close();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        String[] lineArray = sb.toString().toLowerCase().replaceAll("[^a-z]", " ").split(" ");
+        for (String i : lineArray) {
+            i.split(" ");
```

Comment:
try to create a variable instead of using split method four times

Reply:
ok